test-utils/src/main/kotlin/Utils.kt

```diff
@@ -50,10 +50,12 @@ fun patchExpectedResult(result: File, custom: Pair<String, String>? = null, defi
             .replaceIfNotNull("<REPLACE_URL_PROCESSED>", urlProcessed)
 }
 
-fun patchActualResult(result: String, patchStartAndEndTime: Boolean = false): String {
+fun patchActualResult(result: String, patchDownloadStartEndTime: Boolean = false): String {
```

Comment:
What about using a separate parameter for patching the download time? The name is a bit ambiguous now.

Reply:
Done.